index.html

```diff
@@ -1900,7 +2018,9 @@ <h3>Choosing character encodings</h3>
 </ul>
 </aside>
 
-
+    <div class="req" id="char_string_no_legacy">
+	<p class="advisement">Specifications SHOULD NOT add or define support for <a>legacy character encodings</a> unless there is a specific reason to do so.</p>
```

Comment:
In Encoding we require everything new uses UTF-8:

> Therefore, for new protocols and formats, as well as existing formats deployed in new contexts, this specification requires (and defines) the UTF-8 encoding.

Reply:
Good callout. This section needs to be redone, not just patched.